tools/moon-key.ts

```diff
@@ -0,0 +1,29 @@
+import { mnemonicToSeedSync, generateMnemonic } from "bip39";
+import { hdkey } from "ethereumjs-wallet";
+import yargs from "yargs";
+
+const argv = yargs(process.argv.slice(2))
+  .usage('Usage: $0 [--mnemonic "..."] [--account-index x]')
+  .version("1.0.0")
+  .options({
+    mnemonic: { type: "string" },
+    "account-index": { type: "number", default: 0 },
+    strength: { type: "number", default: 256 },
+  }).argv;
+
+const account_index = argv["account-index"];
+const strength = argv["strength"];
+const mnemonic = argv["mnemonic"] || generateMnemonic(strength);
+
+const main = async () => {
+  const hdwallet = hdkey.fromMasterSeed(mnemonicToSeedSync(mnemonic));
+  const path = `m/44'/60'/0'/0/${account_index}`;
```

Comment:
Why this path? I would either let the user choose the derivation path or choose a simpler one.

Reply:
I believe this is the common path used for Ethereum following https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#path-levels

Reply:
Oh cool. I never knew that.